xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -956,7 +956,7 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 			<xsd:element name="TactileActuators" type="xsd:boolean" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Whether Lift has a tactile actuator.</xsd:documentation>
+					<xsd:documentation>Whether there are buttons to start the TRAVELATOR if it is not moving, and whether these buttons are intended and suited for the visually impaired.</xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>Whether there are buttons to start the LIFT if it is not moving, and whether these buttons are intended and suited for the visually impaired.</xsd:documentation>
```

Reply:
Given that in the *LiftButtonsGroup* we have *BrailleButtons*, *RaisedButtons*, and *TactileGroundFloorButton*, then the only meaning that seems to make sense for me would be something like: "Whether at least the call buttons have tactile markers for the visually impaired." OR **"Whether all buttons have tactile markers suited for the visually impaired."** The second one sounds better to me, but I don't know what the initial intention was.